modules/authorization/proc-enabling-the-rbac-plugin.adoc

```diff
@@ -39,6 +39,24 @@ permission:
       users:
         - name: user:default/__<your_policy_administrator_name>__
 ----
+. In order for the {product-short} Web UI to display available permissions provided by installed plugins, the corresponding plugin
+IDs must be added to your custom {product-short} ConfigMap, such as `app-config-rhdh`.
```

Comment:
Consider avoiding the passive voice, and be one of the first mmodule to use the new attribute and link to the _Configuring_ guide:

```suggestion
. In order for the {product-short} Web UI to display available permissions provided by installed plugins, add the corresponding plugin IDs to link:{configuring-book-url}[your custom `{my-app-config-file}` {product-short} configuration file].
```